modules/fuse-box-responsive-api/index.js

```diff
@@ -9,10 +9,11 @@
 
     /* @if universal */
     var isBrowser = typeof window !== "undefined";
+    /* @if !npm */
     if (!isBrowser) {
```

Comment:
Exactly my point, target : `npm` and `client` and `universal` are incompatible. Publishing will npm will preserve require statements, which will handled accordingly by a bundling tool.